package.json

```diff
@@ -10,7 +10,8 @@
     "dist"
   ],
   "scripts": {
-    "test": "tsc --noEmit && vitest --run",
+    "test": "vitest --run",
+    "test:watch": "vitest --watch",
```

Comment:
This change is not bad, but we want to be careful about changing `package.json`, so we should not introduce it in this PR.

Reply:
@yusukebe will create a separate PR for this change, but i know vitest, that's why i made this change :)